.eslintignore

```diff
@@ -7,3 +7,4 @@ build/**
 packages/**/*
 tests/qunit/setup/tinymce.js
 /tmp/**
+vendor_prefixed/
```

Comment:
do we need copy of vendor?

Reply:
@Nevoss do you have a suggestion I am open to it